src/scripts/macros/rest-for-the-night.ts

```diff
@@ -113,7 +113,10 @@ export async function restForTheNight(options: ActionDefaultOptions): Promise<Ch
         const withFrequency = actionsAndFeats.filter(
             (a) =>
                 a.frequency &&
-                (a.frequency.per === "day" || Duration.fromISO(a.frequency.per) <= Duration.fromISO("PT8H")) &&
+                tupleHasValue(
+                    ["turn", "round", "day", Duration.fromISO(a.frequency.per) <= Duration.fromISO("PT8H")],
```

Comment:
the duration check should be separate from the tupleHasValue, tupleHasValue is basically an array includes() but in a way that satisfies typescript.

The check that's occurring here, since its not documented, is that we want to reset all frequencies that are either daily prep or faster than nap-time. But turn/round are encounter mode timespans and don't correspond to fix time exactly (outside of the 6 second nonesense) so you have to explicitly check for those alongside day.